airbyte-config/models/src/main/resources/types/JobSyncConfig.yaml

```diff
@@ -39,6 +39,10 @@ properties:
   destinationDockerImage:
     description: Image name of the destination with tag.
     type: string
+  sourceResourceRequirements:
+    "$ref": ActorDefinitionResourceRequirements.yaml
```

Comment:
I'm a bit confused - where is this `ActorDefinitionResourceRequirements.yaml` file defined? I don't see it being added in this PR

Reply:
Ah nevermind, I just saw that it is being added in this PR: https://github.com/airbytehq/airbyte/pull/10481/files
Looks like I was reviewing these in the wrong order